Data curation with configure parameter `query_group` (#327)
aiidateam/aiida-optimade: README.md

To learn more about this, see the [`optimade-python-tools`](https://github.com/Materials-Consortia/optimade-python-tools) repository.

> ### Using AiiDA group for data curated

Reviewer: ```suggestion
### Using AiiDA group for curated data
```

> The [AiiDA group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curated data and server in bias. 

Reviewer: I don't understand what you mean by "and server in bias".
```suggestion
An [AiiDA Group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curate data and serve only this curated data through the OPTIMADE server. 
```

> An [AiiDA Group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curate data and server only this curated data through the OPTIMADE server.

Reviewer: ```suggestion
An [AiiDA Group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curate data and serve only this curated data through the OPTIMADE server.
```

Author: Ha, it was changed back after I did the rebase 😅 Done

> By setting the `query_group` in `config.json` can only server the structure data from the group.

Reviewer: ```suggestion
Setting the `query_group` option in `config.json` will ensure only the valid (`StructureData`, `CifData`) data nodes in the given AiiDA Group will be served.
```

> Set the `query_group` parameter to `null` (default) to server all structure data from the database.

Reviewer: ```suggestion
Set the `query_group` parameter to `null` (default) to serve all structure data from the database.
```
